src/bin/sol-fbp-generator/main.c

```diff
@@ -1886,6 +1886,9 @@ print_usage(const char *program)
         "    -j  When resolving types, use the passed DESC files. If DESC is\n"
         "        a directory then all the .json files in the directory will be used.\n"
         "        Multiple -j can be passed.\n"
+        "        Note that Soletta install dir will also be looked, as well as any\n"
+        "        path on SOL_FLOW_DESCRIPTIONS_PATHS (where multiple paths are\n"
```

Comment:
Maybe SOL_FLOW_DESCRIPTION_PATH instead of PATH (all the other environment variables I know use singular even when they are a list).